File: Core/EASY/Problem-268/Python/missingNumber.py
from typing import *
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        xor1 = 0
        xor2 = 0
        for i in range(len(nums)):
            xor1 ^= i
            xor2 ^= nums[i]
        xor1 ^= len(nums)
        return xor1 ^ xor2

File: Core/EASY/Problem-268/Java/missingNumber.java
class Solution {
    public int missingNumber(int[] nums) {
        int xor1 = 0;
        int xor2 = 0;
        for(int num = 0 ; num < nums.length; num++){
            xor1 ^= num;
            xor2 ^= nums[num];
        }
        xor1 ^= nums.length;
        return xor1 ^ xor2;
    }
}